ludwig/features/audio_feature.py

```diff
@@ -92,7 +92,9 @@ def cast_column(column, backend):
         return column
 
     @staticmethod
-    def get_feature_meta(column, preprocessing_parameters: PreprocessingConfigDict, backend):
+    def get_feature_meta(
+        column, preprocessing_parameters: PreprocessingConfigDict, backend, **kwargs
```

Comment:
Use `is_input_feature` explicitly instead of `kwargs`, which requires higher cognitive load when trying to understand the code.